src/Meshes/mesh.ts

```diff
@@ -1332,7 +1332,7 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
      * @param applyMorph  defines whether to apply the morph target before computing the bounding info
      * @returns the current mesh
      */
-    public refreshBoundingInfo(applySkeleton: boolean = false, applyMorph: boolean = true): Mesh {
+    public refreshBoundingInfo(applySkeleton: boolean = false, applyMorph: boolean = false): Mesh {

```

Comment:
Also a breaking change for `applyMorph`, but probably more a bug as I don't see why it was `true` by default previously...

Reply:
Yep same

Reply:
I believe this was added in 5.0 so should be okay.